JustSaying/Messaging/MessageProcessingStrategies/IMessageProcessingStrategy.cs

```diff
@@ -21,7 +22,8 @@ public interface IMessageProcessingStrategy
         /// Launch a worker to start processing a message.
         /// </summary>
         /// <param name="action"></param>
-        void StartWorker(Func<Task> action);
+        /// <param name="ct">The cancellation token</param>
+        void StartWorker(Func<Task> action, CancellationToken ct);
 
```

Comment:
We must remember to document this in a change log or release note

Reply:
Yeah, I’ll make a note of all the breaking changes in this PR once it’s merged

Reply:
This should have the proper name of `cancellationToken` if it’s public, rather than just `ct`.

Reply:
Renamed @martincostello